tests/baselines/reference/genericCallWithNonSymmetricSubtypes.errors.txt

```diff
@@ -1,48 +0,0 @@
-tests/cases/conformance/types/typeRelationships/typeInference/genericCallWithNonSymmetricSubtypes.ts(12,9): error TS2453: The type argument for type parameter 'T' cannot be inferred from the usage. Consider specifying the type arguments explicitly.
-  Type argument candidate '{ x: number; y?: number; }' is not a valid type argument because it is not a supertype of candidate '{ x: number; z?: number; }'.
-    Property 'y' is missing in type '{ x: number; z?: number; }'.
-tests/cases/conformance/types/typeRelationships/typeInference/genericCallWithNonSymmetricSubtypes.ts(13,10): error TS2453: The type argument for type parameter 'T' cannot be inferred from the usage. Consider specifying the type arguments explicitly.
-  Type argument candidate '{ x: number; z?: number; }' is not a valid type argument because it is not a supertype of candidate '{ x: number; y?: number; }'.
-    Property 'z' is missing in type '{ x: number; y?: number; }'.
-
-
-==== tests/cases/conformance/types/typeRelationships/typeInference/genericCallWithNonSymmetricSubtypes.ts (2 errors) ====
-    // generic type argument inference where inference leads to two candidates that are both supertypes of all candidates
-    // we choose the first candidate so the result is dependent on the order of the arguments provided
-    
-    function foo<T>(x: T, y: T) {
-        var r: T;
-        return r;
-    }
-    
-    var a: { x: number; y?: number; };
-    var b: { x: number; z?: number; };
-    
-    var r = foo(a, b); // { x: number; y?: number; };

```

Comment:
I think inferring an intersection of the remaining supertypes would be *very* non-intuitive. For example, it the simplest of cases we'd be telling you that `string` isn't assignable to `string & number`. It would just be a bogus type generator!